hazelcast-client/src/main/java/com/hazelcast/client/spi/ProxyManager.java

```diff
@@ -320,8 +321,10 @@ public void removeProxy(String service, String id) {
     }
 
     private void initializeWithRetry(ClientProxy clientProxy) throws Exception {
-        final long retryCountLimit = getRetryCountLimit();
-        for (int retryCount = 0; retryCount < retryCountLimit; retryCount++) {
+        int invocationTimeoutSeconds = getInvocationTimeoutSeconds();
+        long invocationTimeoutMillis = TimeUnit.SECONDS.toMillis(invocationTimeoutSeconds);
```

Comment:
why not change getInvocationTimeoutSeconds to getInvocationTimeoutMilliseconds and do the conversion in the method?